Xamarin.Forms.Platform.UAP/LabelRenderer.cs

```diff
@@ -152,6 +157,34 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 			base.OnElementPropertyChanged(sender, e);
 		}
 
+		void UpdateTextDecorations()
+		{
+			var elementTextDecorations = Element.TextDecorations;
+			if (Element.IsSet(Label.TextDecorationsProperty))
+			{
+				if (_underlineSet)
+				{
+					Control.TextDecorations = Control.TextDecorations & ~Windows.UI.Text.TextDecorations.Underline;
```

Comment:
This part has me a little concerned. Is this basically saying that whenever UpdateTextDecorations is called, it just flips both underline and strikethrough at the same time, back and forth? Shouldn't you need to check if the TextDecoration has actually been removed first?

Reply:
_underlineSet and _strikeThroughset are for backwards compatibility, I don't want to remove a strikethough or underline unless someone has explicitly set it using the new BP.

Youre right that what I'm doing is clearing both flags every time, but then I'm adding back the ones that are present on the BP -- so if one flag is removed it won't be added back.

Reply:
Unless I'm misunderstanding something `& ~` doesn't flip the bit, it sets it to 0 if it is 1. (`^=` flips the bit).

Reply:
I do realize now though that it would be more concise to do `Control.TextDecorations &= ~Flag`, so I'll go ahead and change that for now.

Reply:
okay I must have just been looking at this code for too long -- I understand your comments now and will make it so that these are only removed if they aren't on the BP, not sure how I missed this.